from ESSArch_Core.exceptions import ESSArchException


class UnknownValidator(ESSArchException):
    pass
